README.md

```diff
@@ -417,3 +417,16 @@ netrc-file = /PATH/TO/.netrc
 GITHUB_TOKEN=$YOUR_GITHUB_TOKEN niv add ...
 ```
 
+### How do I import a subpath of a source?
```

Comment:
Maybe also include how this can be used for NixOS modules, part from that it looks good to me.

Reply:
Ah! And how are they used? :stuck_out_tongue:

Reply:
```nix
let 
  sources = import ./nix/sources.nix;
in {
  imports = [ "${sources.package}/path/to/module" ];
}
```

Reply:
Added!